### DISCUSSION TITLE
pyannote-audio 2.0 Release

### DISCUSSION THREAD

**Discussion by mogwai** [pyannote/pyannote-audio#659] [Development]
@hbredin What do you see as the final things needed before you can make the 2.0 the master branch?

**Comment by hbredin** [comment]
Here is a (probably non-exhaustive) list of things that are still missing:

- [x] A good speaker embedding model
- [ ] A good speaker diarization pipeline based on 
  - [pyannote/segmentation](https://huggingface.co/pyannote/segmentation) segmentation model
  - [pyannote/embedding](https://huggingface.co/pyannote/embedding) speaker embedding model
- [ ] A proper documentation of
  - [ ] the `pyannote-audio-train` CLI
  - [ ] the Python API
- [ ] Tutorials
  - [ ] Update tutorials from `master` 
  - [ ] Create _how to add your own task_ tutorial
  - [ ] Create _how to add your own model_ tutorial

**Reply by signofthefour** [reply]
Interesting work, I still wait for a better model of speaker embedding, which provided a bad result (vectors are quite similar (small cosine distance) between male and female voice ) in this version. Thank you!

**Comment by hbredin** [comment]
Work on *Add your own task* tutorial happens here: https://github.com/pyannote/pyannote-audio/pull/665

**Reply by hbredin** [reply]
A [decent speaker embedding](https://huggingface.co/pyannote/embedding) has been released.
